### ISSUE TITLE
[T2][Accessibility][Functional] Social/History tab: Unable to navigate to different categories

### ISSUE THREAD

**Issue by Tanushekhawat** [Tanushekhawat/Tanu#18]
Social/History tab: Unable to navigate to different categories
